docs/GraphQL.md

```diff
@@ -0,0 +1,755 @@
+# GUAC GraphQL definitions
+
+This documents provides some insight into how the GraphQL API matches the [GUAC
+ontology](https://github.com/guacsec/guac/blob/main/docs/ontology.md) and its
+[definition](https://github.com/guacsec/guac/blob/main/docs/ontology-definitions.md).
+
+Note: the GraphQL definitions are not yet stable, they might change in future
+code changes. To get an up to date view of the definitions, you can use one of
+the following:
+
+- Directly consult the source of truth represented by the [GraphQL SDL
+  schema](https://github.com/guacsec/guac/tree/main/pkg/assembler/graphql/schema).
+- Look at the generated Go documentation using `godoc` and analyzing the
```

Comment:
can we link an alternate link to the public godoc?

Reply:
Looks like we need to do that after the next release.